apps/data-types.md

```diff
@@ -495,10 +495,27 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 
 | Attribute | Type | Description |
 | - | :-: |  - |
-| createdAfter | string | The date and time of orders created after this timestamp (format according to ISO 8601). |
-| createdBefore | string | The date and time of orders created before this timestamp (format according to ISO 8601). |
+| createdAfter | string | The date/time of orders created after this timestamp. Expressed according to ISO 8601. Example: `2017-02-04T08:42:49.000Z` |
+| createdBefore | string | The date/time of orders created before this timestamp. Expressed according to ISO 8601. Example: `2017-02-04T08:42:49.000Z` |
 | salesPerCurrency | array of [sales](page:apps-data-types#sales)| The sales summary grouped by currency. |
 
+## scriptTag
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| scriptTagId | string | The unique identifier for the script tag. |
+| scriptUrl | string | The location of the script tag. |
+| scriptType | string | Identifies the scripting language of code embedded within a script tag. |
```

Comment:
no ;-) it is simply the type of the script and could be used in the future.
It can be ignored by developers but we should mention it in the docu that this attribute will be returned.